lola.html

```diff
@@ -0,0 +1,953 @@
+      <ul>
+        <li>Request a destination server to copy an ActivityPub account from a source server</li>
+        <li>Authorize the destination server to the source server</li>
+        <li>See the content in its new location after the destination server completes copying it over</li>
+        <li>Optionally, at a later time, ask the source server to send notifications to followers that the account is moving </li>
+        <li>Optionally, at a later time, redirect the content at the source server to the destination server</li>
+      </ul>
+
+      <p>The approach taken in this document is a set of normatively-described resources, along with a non-normatively
+        described over-arching process. The described out in this document are thus not normative - they are 
+        both a way of breaking down the content in this doc and a way of guiding implementors towards a process that
+      should work for moving accounts.  No shared state is implied in the phases.</p>
+
+      <p>Some of the pieces required to complete this workflow do not need to described as protocol or schema in order to be interoperable.  For example, the user can interact with an ActivityPub server using its Web pages for account management; for these tasks we need to describe what the server ought to be able to do, not how the user can ask it what to do.  This specification uses the language of a <i>user communicating intent to a server</i>, 
+      and whether this intent is communicated via a Web page or a client protocol is out of scope of this document.</p>
+
+      <p>The server-to-server communications involved in this workflow do need to be reliable (URLs, messages and schemas) in order to be interoperable, so those sections are normative.</p>
+
+    </section>
+
+
+    <section id="sotd">
+      <p>This document is an individual submission to the SocialWeb Incubator Community Group, which has formed a task force to focus on the work of data portability.  Interested parties should join the public-swicg@w3.org mailing list to discuss.</p>
+
+      <p>This document is compatible with the ActivityPub Data Portability document, which describes a way to perform an offline account migration.</p>
+
+    </section>
+
+    <section class="informative">
+      <h2>Approach</h2>
+      <p>The technical approach proposed for LOLA is, at a high level: </p>
+      <ul>
+        <li>Source server advertises an OAuth endpoint for authorizing account portability (this can co-exist with other OAuth endpoints e.g.  for authenticating the user).</li>
+        <li>Destination server initiates OAuth to gain user authorization and give the destination server a secure token for source server access requests</li>
+        <li>Destination server can use the secure token and find the right endpoints to start copying data from the source</li>
+        <li>The focus is on copying content (rather than a changelog).  The destination may populate an ActivityPub outbox.</li>
```

Comment:
```suggestion
        <li>The focus is on copying current account content (rather than a changelog or delta).  The destination may populate an ActivityPub outbox that mirrors or subsets the old outbox.</li>
```